mteb/models/model_implementations/colqwen_models.py

```diff
@@ -125,6 +259,69 @@ def __init__(
     citation=COLPALI_CITATION,
 )
 
+TOMORO_TRAINING_DATA = {
+    # from https://huggingface.co/datasets/vidore/colpali_train_set
+    "vdr-multilingual-train",
+    "colpali_train_set",  # as it contains PDFs
```

Comment:
Update annotation in https://github.com/embeddings-benchmark/mteb/pull/3636

```suggestion
    "VDRMultilingualRetrieval",
    # from https://huggingface.co/datasets/vidore/colpali_train_set
    "VidoreDocVQARetrieval",
    "VidoreInfoVQARetrieval",
    "VidoreTatdqaRetrieval",
    "VidoreArxivQARetrieval",
```